metadata-ingestion/tests/unit/test_sql_lineage_parser_impl.py

```diff
@@ -0,0 +1,20 @@
+from datahub.utilities.sql_lineage_parser_impl import SqlLineageSQLParserImpl
+
+
+def test_hash_in_sql_query_with_no_space():
+    parser = SqlLineageSQLParserImpl(
+        sql_query="""
+/*
+HERE IS A STANDARD COMMENT BLOCK
+THIS WILL NOT BREAK sqllineage
+*/
+CREATE OR REPLACE TABLE `foo.bar.trg_tbl`AS
+#This, comment will break sqllineage
```

Comment:
I'm assuming that this was the earlier behavior. Even this comment would not break it now, is that right?